File: docs/components_page/components/popover/popover.R
library(dashBootstrapComponents)

popover_children <- "I am a popover!"

popovers <- div(
  list(
    dbcButton(
      "Click",
      id = "click-target", n_clicks = 0,
      color = "danger", className = "me-1"
    ),
    dbcPopover(
      popover_children,
      target = "click-target",
      body = TRUE,
      trigger = "click"
    ),
    dbcButton(
      "Focus",
      id = "focus-target", n_clicks = 0,
      color = "danger", className = "me-1"
    ),
    dbcPopover(
      popover_children,
      target = "focus-target",
      body = TRUE,
      trigger = "focus"
    ),
    dbcButton(
      "Hover",
      id = "hover-target", n_clicks = 0,
      color = "danger", className = "me-1"
    ),
    dbcPopover(
      popover_children,
      target = "hover-target",
      body = TRUE,
      trigger = "hover"
    ),
    dbcButton("Legacy",
      id = "legacy-target", n_clicks = 0,
      color = "danger"
    ),
    dbcPopover(
      popover_children,
      target = "legacy-target",
      body = TRUE,
      trigger = "legacy"
    )
  )
)

File: docs/components_page/components/popover/popover.jl
using DashBootstrapComponents, DashHtmlComponents

popover_children = "I am a popover!";

popovers = html_div([
    dbc_button(
        "Click",
        id = "click-target",
        color = "danger",
        className = "me-1",
        n_clicks = 0,
    ),
    dbc_popover(popover_children, target = "click-target", trigger = "click"),
    dbc_button(
        "Focus",
        id = "focus-target",
        color = "danger",
        className = "me-1",
        n_clicks = 0,
    ),
    dbc_popover(popover_children, target = "focus-target", trigger = "focus"),
    dbc_button(
        "Hover",
        id = "hover-target",
        color = "danger",
        className = "me-1",
        n_clicks = 0,
    ),
    dbc_popover(popover_children, target = "hover-target", trigger = "hover"),
    dbc_button("Legacy", id = "legacy-target", color = "danger", n_clicks = 0),
    dbc_popover(popover_children, target = "legacy-target", trigger = "legacy"),
]);
